### ISSUE TITLE
Policy extension source

### ISSUE THREAD

**Issue by difo** [GoogleChrome/chrome-extensions-samples#717]
I'm trying to add an extension from a custom repository on azure blob but the error below pops up every time.
![image](https://user-images.githubusercontent.com/6221667/170448169-c8867a1b-02cd-4296-be5d-ed63fc986192.png)

I've also try to force the origin (also any origin) as trusted source.
I also tried to add the extension id to the allowlist
![image](https://user-images.githubusercontent.com/6221667/170448743-8f01ff98-8cc0-4dc2-95c7-4e0385c74095.png)

To be honest it seems that the policy "extensioninstallsources" is ignored also if it's correctly loaded (see status).

We have a strong need to distribute the extension internally without publishing it in the store.

**Comment by guest271314** [comment]
Have you tried chrome://extensions, set Developer mode to on, Load unpacked?

**Comment by difo** [comment]
ciao,
yes in dev mode it works. It's a topic of whitelist a source that is not possible to do neither adding it using the policy that was designed for that.

**Comment by difo** [comment]
yes it's a chrome error when I try to add the extension it pop ups this
![image](https://user-images.githubusercontent.com/6221667/172625264-00ef1bdb-9c68-4fbb-a4e1-e7534ec8153f.png)
also if the policy is correctly loaded see the picture in the very first comment that the policy is displayed with an OK and yes the policies name are "allowlist" "extensioninstallsources"

**Comment by guest271314** [comment]
Then it is a Chrome bug.

**Comment by kunal0x00** [comment]
steps to fix this error : 
step 1: open  >>Properties>>Shortcut>>Target 
step 2: In target section add this line after giving a space  -enable-easy-off-store-extension-install
for example, my Target of chrome is "C:\Program Files\Google\Chrome\Application\chrome.exe" then give one space and add 
'-enable-easy-off-store-extension-install' this line 

Old target:- "C:\Program Files\Google\Chrome\Application\chrome.exe"
Updated target : "C:\Program Files\Google\Chrome\Application\chrome.exe" -enable-easy-off-store-extension-install
